src/main/java/com/softserve/academy/studhub/service/impl/TagServiceImpl.java

```diff
@@ -0,0 +1,56 @@
+package com.softserve.academy.studhub.service.impl;
+
+import com.softserve.academy.studhub.entity.Tag;
+import com.softserve.academy.studhub.repository.TagRepository;
+import com.softserve.academy.studhub.service.TagService;
+import org.springframework.stereotype.Service;
+
+import java.util.List;
+import java.util.Optional;
+
+@Service
+public class TagServiceImpl implements TagService {
+    private final TagRepository tagRepository;
+
```

Comment:
You probably forgot @Autovired

Reply:
It isn't necessary when bean has only one constructor